mappings/net/minecraft/client/render/RenderPhase.mapping

```diff
@@ -37,6 +37,55 @@ CLASS net/minecraft/class_4668 net/minecraft/client/render/RenderPhase
+	FIELD field_29404 ENTITY_CUTOUT_NONULL_OFFSET_Z_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29405 ITEM_ENTITY_TRANSLUCENT_CULL_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29406 ENTITY_TRANSLUCENT_CULL_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29407 ENTITY_TRANSLUCENT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29408 ENTITY_SMOOTH_CUTOUT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29409 BEACON_BEAM_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29410 ENTITY_DECAL_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29411 ENTITY_NO_OUTLINE_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29412 ENTITY_SHADOW_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29413 ENTITY_ALPHA_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29414 EYES_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29415 ENERGY_SWIRL_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29416 LEASH_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29417 WATER_MASK_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29418 OUTLINE_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29419 ARMOR_GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29420 ARMOR_ENTITY_GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29421 TRANSLUCENT_GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29422 GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29423 DIRECT_GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29424 ENTITY_GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29425 DIRECT_ENTITY_GLINT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29426 CRUMBLING_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29427 TEXT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29428 TRANSPARENT_TEXT_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29429 LIGHTNING_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29430 TRIP_WIRE_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29431 END_PORTAL_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29432 END_GATEWAY_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29433 LINES_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29434 NO_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29435 BLOCK_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29436 NEW_ENTITY_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29437 POSITION_COLOR_LIGHTMAP_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29438 POSITION_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29439 POSITION_COLOR_TEX_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29440 POSITION_TEX_SHADER Lnet/minecraft/class_4668$class_5942;
+	FIELD field_29441 POSITION_COLOR_TEX_LIGHTMAP_SHADER Lnet/minecraft/class_4668$class_5942;
```

Comment:
```suggestion
	FIELD field_29439 POSITION_COLOR_TEXTURE_SHADER Lnet/minecraft/class_4668$class_5942;
	FIELD field_29440 POSITION_TEXTURE_SHADER Lnet/minecraft/class_4668$class_5942;
	FIELD field_29441 POSITION_COLOR_TEXTURE_LIGHTMAP_SHADER Lnet/minecraft/class_4668$class_5942;
```

Reply:
Hmm, in #2136 they are named in a similar fashion. Do we have any precedence on using full names or corrected name order than what's supplied by bad mojang names?